### ISSUE TITLE
This isn't all that good

### ISSUE THREAD

**Issue by Eli-Teamruby** [IOnlyPlayAsDrift/RPANPlaysMinecraft#1]
Hear me out...
I made a new version of your script that is much better and works more efficiently.
Here. I've linked the python3 script below.
You don't have to credit me, but I'd appreciate it if you did.
[rpanwritesastory_butitsbetter.py.zip](https://github.com/IOnlyPlayAsDrift/RPANPlaysMinecraft/files/4565075/rpanwritesastory_butitsbetter.py.zip)

**Comment by IOnlyPlayAsDrift** [comment]
yeah i quit on making rpan writes a story several months ago and i never
streamed it so i have no plans on ever streaming it, but thanks anyways lmao
